automation/terraform/modules/testnet-alerts/templates/testnet-alert-rules.yml.tpl

```diff
@@ -272,6 +272,15 @@ groups:
       description: "The rate of {{ $value }} new blocks observed by archive postgres instances is low on network {{ $labels.testnet }}."
       runbook: "https://www.notion.so/minaprotocol/Archive-Node-Metrics-9edf9c51dd344f1fbf6722082a2e2465"
 
+  - alert: OneOfNodeRestarted
+    expr: min by (testnet) (Coda_Runtime_process_uptime_ms_total ${rule_filter}) < 60000
+    labels:
+      testnet: "{{ $labels.testnet }}"
+      severity: warning
+    annotations:
+      summary: "At least one of the nodes on {{ $lables.testnet }} restarted"
+      description: "{{ $value }} nodes on {{ $labels.testnet }} restarted"
```

Comment:
As written, it seems like this will expand to `{$ms_since_last_node_restarted} nodes on mainnet restarted`, which is probably not the message we want here.

If you make the `count` change suggested above, then this message will make more sense; otherwise you'll probably want to change this to e.g. `{{ $value }} ms since a node restarted`